### PR TITLE
Use EmscriptenApplication for emscripten ports

### PR DESCRIPTION
**Pull request by Squareys**
Hello @mosra !

As part of mosra/magnum#300, here are the adaptions to the emscripten example ports.

Cheers, Jonathan

### CODE REVIEW

**Comment by Squareys** [comment]
Note: With the current state, Chrome tabs crash on text example when loading the font (but works on firefox).

File: src/triangle/TriangleExample.cpp [hunk 1]
```diff
@@ -82,6 +85,7 @@ TriangleExample::TriangleExample(const Arguments& arguments):
 
 void TriangleExample::viewportEvent(ViewportEvent& event) {
     GL::defaultFramebuffer.setViewport({{}, event.framebufferSize()});
+    Debug() << event.framebufferSize();
```
Comment (mosra) [inline src/triangle/TriangleExample.cpp]: A leftover from debugging? Similarly for the Renderer include above.

File: src/imgui/ImGuiExample.cpp [hunk 2]
```diff
@@ -202,7 +204,11 @@ void ImGuiExample::mouseMoveEvent(MouseMoveEvent& event) {
 }
 
 void ImGuiExample::mouseScrollEvent(MouseScrollEvent& event) {
-    if(_imgui.handleMouseScrollEvent(event)) return;
+    if(_imgui.handleMouseScrollEvent(event)) {
+        /* Prevent scrolling the page */
+        event.setAccepted();
+        return;
+    }
```
Comment (mosra) [inline src/imgui/ImGuiExample.cpp]: Can you put this change into #60 instead? Makes sense to have it in master.

**Comment by Squareys** [comment]
> Is that still a thing or not anymore? I remember you said it's gone.

Yeah, that's gone :)

**Comment by Squareys** [comment]
> tabindex comment from #60 applies here as well.

I'll just drop 76b75bc0b726e81d9153c5f1cb9b8a0d79434fcd then 👍

**Comment by Squareys** [comment]
@mosra This is again ready for review!

File: modules/FindMagnum.cmake [hunk 3]
```diff
@@ -67,6 +67,8 @@
 #  TextureTools                 - TextureTools library
 #  Trade                        - Trade library
 #  Vk                           - Vk library
+#  AndroidApplication           - Android application
```
Comment (mosra) [inline modules/FindMagnum.cmake]: I'll update the Find module when merging master to `ports` once #60 is in, so this commit will get ignored anyway.

**Comment by mosra** [comment]
Merged as 3fb7bd8f63aa402ebcfb6e77f172c59467082943, with the imgui change being put directly into master as 4fae2fd391990570f06598aecf5c2cb5985a5593. All the examples are now shipped on https://magnum.graphics/showcase/ and looking & behaving better than ever :)

Thanks a lot!

**Pull request closed without merging**